lib/src/model/container_member.dart

```diff
@@ -25,11 +26,8 @@ mixin ContainerMember on ModelElement implements EnclosedElement {
   }
 
   @override
-  Set<String> get features {
-    var _features = super.features;
-    if (isExtended) _features.add('extended');
-    return _features;
-  }
+  Set<Feature> get features =>
+      {...super.features, if (isExtended) Feature.extended};
```

Comment:
If this will be more readable on multiple lines, a trailing comma will force that.

Reply:
done